docs/content/guides/upgrading.md

```diff
@@ -1,73 +1,123 @@
+
 ## Backup your cluster
 
 We strongly suggest performing a full backup of your Kubernetes cluster, such as storage and etcd.
 Use your favorite tool according to your needs.
 
-## Uninstall the old Capsule release
+## Scale down the Capsule controller
 
-If you're using Helm as package manager, all the Operator resources such as Deployment, Service, Role Binding, and etc. must be deleted.
+Using the `kubectl` or Helm, scale down the Capsule controller manager:
 
 ```
-helm uninstall -n capsule-system capsule 
+helm upgrade -n capsule-system capsule --set "replicas=0" 
 ```
 
-Ensure that everything has been removed correctly, especially the Secret resources.
+> This is required to avoid the old Capsule version from processing objects that aren't yet installed as a CRD.
+>
+> Ensure that all the Pods have been removed correctly.
 
 ## Patch the Tenant custom resource definition
 
-Helm doesn't manage the lifecycle of Custom Resource Definitions, additional details can be found [here](https://github.com/helm/community/blob/f9e06c16d89ccea1bea77c01a6a96ae3b309f823/architecture/crds.md).
+Unfortunately, Helm doesn't manage the lifecycle of Custom Resource Definitions, additional details can be found [here](https://github.com/helm/community/blob/f9e06c16d89ccea1bea77c01a6a96ae3b309f823/architecture/crds.md).
+
+> If you're upgrading to v0.2.0, please, pay attention to the section in this document named `CapsuleConfiguration new version`
 
 This process must be executed manually as follows:
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/clastix/capsule/v0.1.0/config/crd/bases/capsule.clastix.io_tenants.yaml
+kubectl apply -f https://raw.githubusercontent.com/clastix/capsule/v0.2.0/config/crd/bases/capsuleconfiguration-crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/clastix/capsule/v0.2.0/config/crd/bases/globaltenantresources-crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/clastix/capsule/v0.2.0/config/crd/bases/tenant-crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/clastix/capsule/v0.2.0/config/crd/bases/tenantresources-crd.yaml
 ```
 
 > Please note the Capsule version in the said URL, your mileage may vary according to the desired upgrading version.
 
-## Install the Capsule operator using Helm
+## Update your Capsule Helm chart
+
+Ensure to update the Capsule repository to fetch the latest changes.
+
+```
+helm repo update
+```
+
+The latest Chart must be used, at the current time, 0.3.0 is expected for Capsule v0.2.0, you can fetch the full list of available charts with the following command.
+
+```
+helm search repo -l clastix/capsule
+```
 
 Since the Tenant custom resource definition has been patched with new fields, we can install back Capsule using the provided Helm chart.
 
 ```
-helm upgrade --install capsule clastix/capsule -n capsule-system --create-namespace
+helm upgrade --install capsule clastix/capsule -n capsule-system --create-namespace --version 0.3.0
```

Comment:
We should probably advise users, that we expect by default capsule to be installed in `capsule-system`. If that's not the case they need to manually patch the namespace/service after helm install.

```
CUSTOM_NS="tenancy-operations"

# Patch CapsuleConfiguration
kubectl patch crd capsuleconfigurations.capsule.clastix.io --type='json' -p=" [{'op': 'replace', 'path': '/spec/conversion/webhook/clientConfig/service/namespace', 'value': "${CUSTOM_NS}"}]"

# Patch Tenant
kubectl patch crd tenants.capsule.clastix.io --type='json' -p=" [{'op': 'replace', 'path': '/spec/conv

```
# Patch Tenant
kubectl patch crd tenants.capsule.clastix.io --type='json' -p=" [{'op': 'replace', 'path': '/spec/conversion/webhook/clientConfig/service/namespace', 'value': "${CUSTOM_NS}"}]"
```